File: core/src/main/java/rs/kunpero/vacation/util/VacationUtils.java
package rs.kunpero.vacation.util;

import java.time.LocalDate;
import java.util.List;

public class VacationUtils {

    public static boolean isWithinRange(LocalDate date, LocalDate from, LocalDate to) {
        return !(date.isBefore(from) || date.isAfter(to));
    }

    public static String convertListToString(List<String> list) {
        return String.join(",", list);
    }
}

File: core/src/main/java/rs/kunpero/vacation/util/VacationUtils.kt
package rs.kunpero.vacation.util

import java.time.LocalDate

class VacationUtils {
    companion object {
        @JvmStatic
        fun isWithinRange(date: LocalDate, from: LocalDate, to: LocalDate): Boolean {
            return !(date.isBefore(from) || date.isAfter(to))
        }

        @JvmStatic
        fun convertListToString(list: List<String>?): String {
            if (list.isNullOrEmpty()) {
                return String()
            }
            return list.distinct().joinToString(",")
        }
    }
}